static-analysis/teammates-eslint-es6.yml

```diff
@@ -20,6 +20,15 @@ rules:
     # enforce a maximum line length
     max-len: ["error", 125]
 
+    # disallow continue statements
```

Comment:
Oh nvm, I get what you mean now; the description above is what the rule does, not the change you made to the existing rule